### ISSUE TITLE
Does this support neovim/nvim ?

### ISSUE THREAD

**Issue by ajeeb-kp-keleno** [dawsonbotsford/swim#29]

**Comment by dawsbot** [comment]
✅Yes, this does! I've had my personal computer running with neovim and `swim` for years 🙌 

Neovim supports the `.vimrc` format, but nvim keeps the configuration file in a different place.

> The Nvim config file is named "init.vim", located at:
		Unix		~/.config/nvim/init.vim
		Windows		~/AppData/Local/nvim/init.vim

---

What works with `swim` is to have your `~/.vimrc` file and symlink `~/.config/nvim/init.vim` to point to `~/.vimrc`. I've done this on my personal computer with the following:

⚠️ Backup what's in `$HOME/.config/nvim/init.vim` before running this ⚠️
```sh
 # WARNING: make sure you back this up first
rm -rf "$HOME/.config/nvim/init.vim"
ln -s "$DIR/vimrc" "$HOME/.config/nvim/init.vim"
```

**Comment by ajeeb-kp-keleno** [comment]
I have a small question. Only symlink-ing `~/.config/nvim/init.vim` needed ? What about `~/.vim` directory ?

By the way, I appreciate your work. I really likes it.
Also, I have written a blog post about swim here. 
https://dev.to/ajeebkp23/swim-switch-between-multiple-vimrcs-2g69
Your feedback about my blog is also welcome :+1:

**Comment by dawsbot** [comment]
Wow, incredible! Nobody has done a blog post on this before, I really appreciate that. Here's my questions along with feedback:

1. I'm not sure what `~/.vim` does. I don't have one in my computer and a quick google search didn't help me learn this. What's that directory used for? 

2. In the blog post you say "Unfortunately, it didn't work well for me." can you inform me what went wrong here? Showing people how quick a good installation is will be important for your blog post to be popular

**Comment by ajeeb-kp-keleno** [comment]
I was thinking about having multiple vimrcs before. I was thinking about creating symlinks and keeping folders like you have done with swim project. But, I hadn't enough time to keep and build it as a good script.  But, suddenly, I remembered about your project, and tried it. It just worked.

1. `~/.vim` (ie. `.vim` folder in your `user-home-folder`) is the folder where files are written related to `~/.vimrc`. ie. Pluggins, syntax highlighting etc. goes to this folder as far I can tell. I have used it vim in both `Windows` and `Linux`. Both follow this pattern. Perhaps, you just use plain vim (with out pluggins) !!? [I don't know, I'm taking a wild guess]

2. a) I have suggested few commands to make the installation working. It's just after that statement. It starts with this statement `Below steps, are what I followed`. 
 b) Anyway, the problem was, it was not detecting `swim` binary, during installation and after installation. By the way, I was able to figure out the path issue with your Readme alone. That's good thing. I really happy about it. Congrats.  I managed to fix the path via some methods I mentioned in the blog.

I use Debian Linux. Debian 11 (ie.Testing Release, it's very recent one with almost all latest piece of software, I ever wanted).
